crates/node_binding/package.json

```diff
@@ -28,7 +29,8 @@
     "@napi-rs/cli": "3.0.0-alpha.73",
     "@napi-rs/wasm-runtime": "^0.2.7",
     "emnapi": "^1.3.1",
-    "typescript": "^5.7.3"
+    "typescript": "^5.7.3",
+    "@emnapi/core": "^1.3.1"
```

Comment:
what's this

Reply:
runtime dependency of [emnapi](https://github.com/toyobayashi/emnapi)